tests/unit/models/test_settings.py

```diff
@@ -0,0 +1,22 @@
+from runner_manager.models.settings import Settings
+from pytest import fixture
+
+
+def test_settings_default_values():
+    settings = Settings()
+    assert settings.name == "runner-manager"
+    assert settings.redis_om_url == "redis://localhost:6379/0"
+    assert settings.github_base_url == "http://localhost:4010"
+
+@fixture
+def test_env_settings():
+    env_settings = {
+        "NAME": "name-test",
+        "REDIS_OM_URL": "redis://localhost:6379/1",
+        "GITHUB_BASE_URL": "http://localhost:4999",
+    }
+    settings = Settings()
+    settings.Config.customise_sources(env_settings=env_settings, init_settings={}, file_secret_settings={})
```

Comment:
customize source is implicitly called, no need to make direct call to it, if you want to test env settings you should set env variable like the following:
```python
import os

os.environ['NAME'] = 'name-test'

settings = Settings()

assert os.environ['NAME'] == 'name-test'